airflow/providers/alibabacloud/provider.yaml

```diff
@@ -0,0 +1,52 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+---
+package-name: apache-airflow-providers-alibabacloud
+name: Alibabacloud
+description: |
+    Alibaba Cloud integration (including `Alibaba Cloud <https://www.alibabacloud.com//>`__).
+
+versions:
+  - 2.1.0
+
+additional-dependencies:
+  - apache-airflow>=2.1.0
+
+integrations:
+  - integration-name: Alibaba Cloud OSS
+    external-doc-url: https://www.alibabacloud.com/help/product/31815.htm
+    tags: [alibabacloud]
+
+operators:
+  - integration-name: Alibaba Cloud OSS
+    python-modules:
+      - airflow.providers.alibabacloud.operators.oss_bucket
+      - airflow.providers.alibabacloud.operators.oss_object
```

Comment:
If possible, we should store operators/hooks/sensors in a file with the same name so that the user does not have to guess the name of the file.